modules/guides/pages/transactions.adoc

```diff
@@ -0,0 +1,603 @@
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+[abstract]
+{description}
+
+== Introduction
+
+Couchbase transactions enable you to carry out ACID (atomic, consistent, isolated, and durable) actions on the database.
+This how-to guide covers N1QL support for Couchbase transactions.
+Some SDKs also support Couchbase transactions.
+Refer to <<related-links>> for further details.
+
+Only DML (data modification language) statements are permitted within a transaction:
+xref:n1ql:n1ql-language-reference/insert.adoc[INSERT],
+xref:n1ql:n1ql-language-reference/upsert.adoc[UPSERT],
+xref:n1ql:n1ql-language-reference/delete.adoc[DELETE],
+xref:n1ql:n1ql-language-reference/update.adoc[UPDATE],
+xref:n1ql:n1ql-language-reference/merge.adoc[MERGE],
+xref:n1ql:n1ql-language-reference/select.adoc[SELECT],
+xref:n1ql:n1ql-language-reference/execfunction.adoc[EXECUTE FUNCTION],
+xref:n1ql:n1ql-language-reference/prepare.adoc[PREPARE], or
+xref:n1ql:n1ql-language-reference/execute.adoc[EXECUTE].
+
+include::partial$before-you-begin.adoc[tag=body]
+include::partial$query-tools.adoc[tags=body;!thumbs]
+
+include::partial$data-warning.adoc[]
+
+[[settings]]
+== Transaction Parameters
+:lead: The following settings set the transaction parameters for the examples in the <<multiple-statement>> section below.
+
+You can specify various settings and parameters to control how transactions work.
+You can access transaction settings and parameters through the Query Workbench, the cbq shell, or the Query REST API.
+
+[tabs]
+====
+Query Workbench::
++
+--
+To specify parameters for a Couchbase transaction, use the query run-time preferences window.
+
+. Click the cog icon icon:cog[] to display the Run-Time Preferences window.
+
+. To specify the transaction scan consistency, open the *Scan Consistency* drop-down list and select an option.
+
+. To specify the transaction timeout, enter a value in seconds in the *Transaction Timeout* box.
+
+. To specify any other parameters, click the btn:[+] button in the *Named Parameters* section.
+When the new named parameter appears, enter the name in the *name* box and a value in the *value* box.
+
+. Choose btn:[Save Preferences] to save the preferences and return to the Query Workbench.
+
+'''
+
+{lead}
+
+image::transactions-preferences.png['The Run-Time Preferences dialog, with Scan Consistency set to "not_bounded", Transaction Timeout set to "120", and named parameter "durability_level" set to "none"']
+
+① Set *Scan Consistency* to `not_bounded`.
+
+② In the *Named Parameters* section, add a named parameter with *name* set to `durability_level` and *value* set to `"none"` (with double quotes).
+
+③ Set *Transaction Timeout* to `120`.
+--
+
+CBQ Shell::
++
+--
+To specify parameters for a Couchbase transaction, use the `\SET` command.
+
+'''
+
+{lead}
+
+[source,n1ql]
+----
+include::n1ql:example$transactions/multiple.n1ql[tag=settings]
+----
+
+{github}
```

Comment:
the `{github}` wording "view this code in context" doesn't make as much sense for Shell output does it?